requirements.txt

```diff
@@ -0,0 +1,4 @@
+cython>=3.0.0
+numpy>=1.10.0
+greenlet>=3.0.0
+cmake
```

Comment:
Thanks! These should probably match `install_requires` in setup.py?

Reply:
done